web/src/OrganizationEditPage.js

```diff
@@ -255,6 +255,30 @@ class OrganizationEditPage extends React.Component {
             }} />
           </Col>
         </Row>
+        <Row style={{marginTop: "20px"}} >
+          <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
+            {Setting.getLabel(i18next.t("Organization:Languages"), i18next.t("Organization:Languages - Tooltip"))} :
+          </Col>
+          <Col span={22} >
+            <Select virtual={false} mode="tags" style={{width: "100%"}}
+              value={this.state.organization.languages}
+              onChange={(value => {
+                this.updateOrganizationField("languages", value);
+              })} >
+              {[
+                {value: "zh", label: "简体中文"},
```

Comment:
Use exsiting order: `["en", "zh", "es", "fr", "de", "ja", "ko", "ru"]`, not alphabet